src/jobs-worker/FloraJobs/ThirdParties/Hackage/Client.hs

```diff
@@ -37,6 +38,14 @@ listHackageUsers = hackageClient // API.listUsers
 getHackageUser :: Text -> ClientM HackageUserDetailsObject
 getHackageUser username = hackageClient // API.withUser /: username // API.getUser
 
+getPackageTarball :: VersionedPackage -> ClientM ByteString
+getPackageTarball versionedPackage =
+  hackageClient
+    // API.withPackage
+    /: versionedPackage
+    // API.getTarball
+    /: VersionedTarball versionedPackage
+
```

Comment:
@RaoulHC For the sake of keeping things simple I think we should postpone importing tarballs from Hackage (especially because Hackage has a very good CDN and we don't).

Reply:
This is why I've made it a configurable option with the default to be off for now, the public server definitely shouldn't be hosting tarballs immediately, and the importing of hackage tarballs is also gated by the feature.

But I think it's worth keeping if only for a way to do a good smoke test, you can load the cabal files of a single package, import the tarballs in the admin menu and with a bit of bash fu check that all the imported tarballs have all the files and are the expected size (or at least roughly, with no guarantee to be the same order they might be marginally different due to compression).

Reply:
ok I see :)